Linux builds are failing

It appears the linux builds stopped working on Oct 21.

There is a warning in the build logs:
```
[warning]An image label with the label ubuntu-16.04 does not exist.
```
I think we just need to update the [`vmImage`](https://github.com/omnia-md/conda-dev-recipes/blob/master/.azure-pipelines/azure-pipelines-linux.yml#L4) to `ubuntu-18.04`.

The builds are running again, but failing to upload with this error:
```
Using authentication token starting with om-b ...
Uploading package '/home/conda/build_artifacts/linux-64/openmm-7.6-py36_cuda80_debug_1.tar.bz2' to anaconda user 'omnia-dev'...
Using Anaconda API: https://api.anaconda.org
[DEBUG] Using token from command line args
Using "omnia-dev" as upload username
Processing '/home/conda/build_artifacts/linux-64/openmm-7.6-py36_cuda80_debug_1.tar.bz2'
Detecting file type...
[DEBUG] Testing if conda package ..
[DEBUG] This is a conda package
File type is "conda"
Extracting conda package attributes for upload
Creating package "openmm"
Creating release "7.6"
Uploading file "omnia-dev/openmm/7.6/linux-64/openmm-7.6-py36_cuda80_debug_1.tar.bz2"
Distribution already exists. Please use the -i/--interactive or --force or --skip options or `anaconda remove omnia-dev/openmm/7.6/linux-64/openmm-7.6-py36_cuda80_debug_1.tar.bz2
[ERROR] 
Traceback (most recent call last):
  File "/opt/conda/bin/anaconda", line 10, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/binstar_client/scripts/cli.py", line 160, in main
    description=__doc__, version=version)
  File "/opt/conda/lib/python3.7/site-packages/binstar_client/scripts/cli.py", line 139, in binstar_main
    return args.main(args)
  File "/opt/conda/lib/python3.7/site-packages/binstar_client/commands/upload.py", line 343, in main
    args=args)
  File "/opt/conda/lib/python3.7/site-packages/binstar_client/commands/upload.py", line 244, in upload_package
    channels=args.labels, callback=upload_print_callback(args))
  File "/opt/conda/lib/python3.7/site-packages/binstar_client/__init__.py", line 520, in upload
    self._check_response(res)
  File "/opt/conda/lib/python3.7/site-packages/binstar_client/__init__.py", line 227, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.Conflict: ('file linux-64/openmm-7.6-py36_cuda80_debug_1.tar.bz2 already exists for package openmm version 7.6', 409)
Retrying after 1 seconds...
```
Here is [one example](https://dev.azure.com/OmniaMD/conda-dev-recipes/_build/results?buildId=718&view=logs&j=9a864fd9-6c8f-52ca-79ce-2aa6dca1a1de&t=c1052e73-18ac-51e7-1f22-d9a958a44bcd&l=10368).

--- a/recipes/openmm/meta.yaml
+++ b/recipes/openmm/meta.yaml
@@ -14,7 +14,7 @@
 
 package:
   name: openmm
-  version: 7.6
+  version: 7.7
 
 source:
   git_url: https://github.com/openmm/openmm.git

